primer-android/src/org/primer/fragment/hot/OptionHotFragment.java

```diff
@@ -400,8 +384,9 @@ public String getOptionNote(int index) {
         }
 
         private String getFeeStr(PrimerjSettings.TransactionFeeMode transactionFeeMode) {
-            float dividend = 100000;
-            String unit = "mXPM/kb";
+            // Set unit to mXPM/b from satoshi/kb and adjust dividend accordingly by a factor of 10^8
+            float dividend = 100000000;
+            String unit = " mXPM/b";
             float fee = (float) transactionFeeMode.getMinFeeSatoshi() / dividend;
```

Comment:
Here the getMinFeeSatoshi() return value is changed from 1,000,000 to 100,000,000 correct?

Reply:
This code is hard to read, maybe consider:
1) get rid of the 'dividend' variable, not sure what it means and why it is named as such
2) what's the difference with MinFee and BaseFee
3) If baseFee is in the unit of Satoshi/KB, then the conversion might read /1000/100,000 and note in the comment that it's for the unit conversion from Satoshi/KB to mXPM/B.
4) unit name better use mXPM/B and Satoshi/KB as in upper case, since sometimes b refers to bit so to avoid possible confusion.